packages/crypto/src/mnemonic/toKeyPair.ts

```diff
@@ -22,12 +22,14 @@ import { SIDETREE_BIP44_COIN_TYPE } from '../constants';
 export const toKeyPair = async (
   mnemonic: string,
   index: number,
-  type = 'Ed25519'
+  type = 'Ed25519',
+  hdPath?: string
```

Comment:
hd path and index are redundant... there should be a warning if you provide both.

ideally we would eventual paramaterize this function with an or type that made this explicit.